include/drivers/ufs.h

```diff
@@ -214,6 +214,9 @@
 #define DESC_TYPE_INTERCONNECT		0x04
 #define DESC_TYPE_STRING		0x05
 
+#define DESC_DEVICE_MAX_SIZE		0x1F
+#define DEVICE_DESC_PARAM_MANF_ID 	0x18
```

Comment:
Style check error - ensure no spaces before tabs; please check that there are only tabs here.